UI/obs-app.cpp

```diff
@@ -1720,7 +1720,9 @@ QString OBSTranslator::translate(const char *context, const char *sourceText,
 				 const char *disambiguation, int n) const
 {
 	const char *out = nullptr;
-	if (!App()->TranslateString(sourceText, &out))
+	QString str(sourceText);
+	str.replace(" ", "");
```

Comment:
You could use `.remove(" ")` but replace works too.

Reply:
I'm OK with either solution. Unsure which is preferred.

Reply:
Me neither.